public/css/lrr.css

```diff
@@ -1,363 +1,438 @@
-    left: 0;
-    top: 0;
-    width: 100%;
-    height: 100%;
-    z-index: 9999;
-    background-color: #000;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 9999;
+  background-color: #000;
 }
 
 .base-overlay {
-    position: fixed;
-    top: 40px;
-    min-height: 70%;
-    z-index: 10000;
-    box-shadow: 0 0 16px #000;
-    padding-top: 10px !important;
-    padding-bottom: 10px !important;
+  position: fixed;
+  top: 40px;
+  min-height: 70%;
+  z-index: 10000;
+  box-shadow: 0 0 16px #000;
+  padding-top: 10px !important;
+  padding-bottom: 10px !important;
 }
 
 .page-overlay {
-    left: 5%;
-    width: 90% !important;
-    overflow-y: scroll;
-} 
+  left: 5%;
+  width: 90% !important;
+  overflow-y: scroll;
+}
 
 .small-overlay {
-    left: 35%;
-    width: 35% !important;
-} 
+  left: 35%;
+  width: 35% !important;
+}
 
 .ie7 .page-overlay {
-    height: 50%;
-} 
+  height: 50%;
+}
 
 #theme-overlay-shade {
-    position: fixed;
-    display: none;
-    left: 0;
-    top: 0;
-    width: 100%;
-    height: 100%;
-    z-index: 9999;
-    background-color: #000;
+  position: fixed;
+  display: none;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 9999;
+  background-color: #000;
 }
 
 .page-number {
-    font-size: 20px; 
-    position: absolute; 
-    top: 40%; 
-    left: 30%; 
-    z-index: -1;
-    padding: 5px;
+  font-size: 20px;
+  position: absolute;
+  top: 40%;
+  left: 30%;
+  z-index: -1;
+  padding: 5px;
 }
 
 a:hover > .page-number {
-    z-index: 300;
-    background-color: #000000;
+  z-index: 300;
+  background-color: #000000;
 }
 
 .reader-image {
-    max-width:100%; 
-    height: auto; 
-    width: auto; 
-    min-height:300px;
-    user-select: none;
+  max-width:100%;
+  height: auto;
+  width: auto;
+  min-height:300px;
+  user-select: none;
 }
 
 /* Option menu. */
 .collapsible {
-    box-shadow: none !important;
+  box-shadow: none !important;
 }
 
-.option-td { 
-    padding-right: 20px;
-    padding-bottom: 10px;
-    vertical-align: top !important;
-    width: 200px;
+.option-td {
+  padding-right: 20px;
+  padding-bottom: 10px;
+  vertical-align: top !important;
+  width: 200px;
 }
 
 .collapsible-title {
-    border: none !important;
+  border: none !important;
 }
 
 .collapsible-body {
-    border: none !important;
+  border: none !important;
 }
 
 .option-flyout > .collapsible-title {
-    font-size: 18px;
-    font-weight: bold;
+  font-size: 18px;
+  font-weight: bold;
 }
 
 .option-flyout > .collapsible-body {
-    padding: 10px !important;
+  padding: 10px !important;
 }
 
 .left-column {
-    display:inline-block; 
-    width: 400px;
-    vertical-align: top;
+  display: inline-block;
+  width: 400px;
+  vertical-align: top;
 }
 
 .right-column {
-    display:inline-block; 
-    width:60%;
+  display: inline-block;
+  width: 60%;
 }
 
 .if a {
-    margin-right: 10px;
+  margin-right: 10px;
+}
+
+li {
+  list-style: none;
```

Comment:
This is new afaik -- Do you remember why you put it there?

Reply:
Yes, that is to get rid of the little dots on `/logs`
<img width="463" alt="Bildschirmfoto 2020-01-23 um 21 08 50" src="https://user-images.githubusercontent.com/6740726/73019839-ae2ab280-3e24-11ea-852a-7f824350ae4a.png">